beta-src/src/models/NodeSetClass.ts

```diff
@@ -0,0 +1,178 @@
+import PathSearchClass from "./PathSearchClass";
+import TerritoryClass from "./TerritoryClass";
+
+export default class NodeSetClass {
+  Nodes: Record<string, TerritoryClass> = {};
+
+  Path: any;
```

Comment:
we should never use any

Reply:
any "any" usage within refactored code is related to convoy path logic.. which I don't fully understand. I believe we agreed on just porting in from legacy code.